x-pack/plugins/spaces/public/views/management/spaces_grid/spaces_grid_page.tsx

```diff
@@ -86,18 +83,21 @@ class SpacesGridPageUI extends Component<Props, State> {
       <Fragment>
         <EuiFlexGroup justifyContent={'spaceBetween'}>
           <EuiFlexItem grow={false}>
-            <EuiText>
+            <EuiTitle size="m">
               <h1>
                 <FormattedMessage
                   id="xpack.spaces.management.spacesGridPage.spacesTitle"
                   defaultMessage="Spaces"
                 />
               </h1>
+            </EuiTitle>
+            <EuiText color="subdued">
+              <p>Spaces allow you to silo content into meaningful categories</p>
```

Comment:
What are your thoughts on using the [existing feature description](https://github.com/mattkime/kibana/blob/mgmt-eui-nav-sidebar/x-pack/plugins/spaces/public/lib/constants.ts#L12) for Spaces here? As an added bonus, it's already i18n-ready